lr1110/lr1110/us_915_ttn.c

```diff
@@ -0,0 +1,32 @@
+#include "lr1_stack_mac_layer.h"
+#include "lr1mac_utilities.h"
+#include "lorawan_api.h"
```

Comment:
Again, what about other regions?

Reply:
It will be hard to test in other regions too